pkg/netlink/netlink.go

```diff
@@ -207,7 +207,7 @@ func DeleteIfaceAndAssociatedRoutes(iface string, tableID int) error {
 	link, err := n.LinkByName(iface)
 	if err != nil {
 		if !errors.Is(err, netlink.LinkNotFoundError{}) {
-			klog.Warningf("Failed to retrieve the vxlan-tunnel interface: %v", err)
+			klog.Warningf("Failed to retrieve the %s interface: %v", iface, err)
```

Comment:
Could you provide this fix and the next as a separate commit in your patchset? That way we can backport it to 0.12 separately from the IP-IP changes.

Reply:
yes - no problem at all